scripts/search_start_points.py

```diff
@@ -0,0 +1,59 @@
+from scipy.optimize import minimize
+import numpy as np
+import os
+import csv
+
+from scripts.models import Qlearning, RescorlaWagner
+from scripts.player import Estimator, RealPlayer
+
+
+def search_optimal_parameters(estimator):
+    optimal_alpha = []
+    optimal_T = []
+    optimal_func_value = []
+    alpha_list = np.arange(0.0, 1.0, 0.01)
+    T_list = np.arange(1, 10, 0.1)
+    for alpha in alpha_list:
+        for T in T_list:
+            optimal_params = minimize(estimator.log_likelihood_function, x0=np.array([T, alpha]), method='Nelder-Mead',
```

Comment:
we should change api of Estimator to make max_log_likelihood reusable so start points could be passed as parameters